programs/golines.nix

```diff
@@ -25,7 +20,7 @@ in
     })
   ];
 
-  config = lib.mkIf cfg.enable {
+  config = lib.mkIf (cfg.enable != null && cfg.enable) {
```

Comment:
Is this not of type bool? How can this be null?

Reply:
If something is not set it doesn't have the value null. So this comparison doesn't work.